extra/gojira-hybrid

```diff
@@ -41,7 +69,16 @@ hybrid_setup() {
   export CLUSTER_KEY_PATH=$HYBRID_PATH/cluster_keys
   export GOJIRA_TARGET
 
-  main "$@"
+  # This only runs on "mode", not in plugin mode. Thus, makes it the plugin
+  # action call
+  case $1 in
+    help)
+      hybrid-usage
+      exit 0
+      ;;
+  esac
+
+  [[ $1 != "plugin" ]] && main "$@"
```

Comment:
This is to keep compatibility with the 'extra command' mode, right?

Reply:
yes. This makes it so hybrid can be both a mode and a plugin